lib/compute/vm.js

```diff
@@ -423,6 +423,42 @@ VM.prototype.reset = function(callback) {
 };
 
 /**
+ * Set the machine type for this instance.
+ *
+ * @resource [Instances: setMachineType API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/instances/setMachineType}
+ *
+ * @param {string} zone - the compute engine zone
```

Comment:
We don't need the zone, since a VM instance has to be accessed through a Zone. It should be available as this.zone.name.